packages/gallery/src/components/item/itemView.js

```diff
@@ -261,16 +261,15 @@ class ItemView extends React.Component {
   shouldShowHoverOnMobile() {
     if (utils.isMobile()) {
       const {
+        controllers_layoutParams_info_allowDescription: allowDescription,
+        controllers_layoutParams_info_allowTitle: allowTitle,
```

Comment:
remove controllers_

Reply:
@shmuelhizmi I removed allowDescription || allowTitle || isStoreGallery params from the condition, they aren't related to the core